app/mentor/models.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from economy.models import SuperModel
+from rest_framework import serializers
+
+
+class Mentor(SuperModel):
+
+    TIME_RANGE = [
+        ('1_5', '1 - 5'),
+        ('5_10', '5 - 10'),
+        ('10_15', '10 - 15'),
+        ('15_20', '15 - 20'),
+        ('20_', '20+'),
+    ]
+
+    profile = models.ForeignKey('dashboard.Profile', related_name='mentor', on_delete=models.CASCADE)
+    name = models.CharField(max_length=255, blank=True)
+    email = models.CharField(max_length=255)
+    org = models.CharField(max_length=255)
+    about = models.CharField(max_length=2000)
+    experience = models.CharField(max_length=255, choices=TIME_RANGE, blank=True)
```

Comment:
You can drop `max_length` here to 5 for the above.  It might be worthwhile to define the experience range types and reference those in a tuple of tuples for choices, so there is always a sure fire way to reference these without guessing what to query by.

Reply:
Ok, i'll add max_length.

Are You talking about defining new choices list for this attribute?